android/AndroidSportsPlatform/app/src/main/AndroidManifest.xml

```diff
@@ -18,6 +18,7 @@
         <activity android:name=".activities.SearchOperationsActivity" />
         <activity android:name=".activities.RegisterActivity" />
         <activity android:name=".activities.ProfileActivity" />
+        <activity android:name=".activities.MainActivity" />
```

Comment:
Changing name _Main_ to _Login_ fits better for login operation.

Reply:
Exactly.